### PR TITLE
finished reviewing tests

### PR DESCRIPTION
**Pull request by ghost**
now moving onto #8

### CODE REVIEW

File: src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package org.dia.algorithms.pdfclustering
```
Comment (chrismattmann) [inline src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala:18]: ALv2 license header

File: src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    val partCount = 2
+
+    val netcdfDir = "resources/merra"
+    /** TSURF(time, lat, lon) */
+    val variables = List("TSURF")
+
+    //    val vars = NetCDFReader.loadNetCDFVar(uri)
+    //    val lon = NetCDFReader.loadNetCDFNDVar(uri, "lon")
+    //    val lat = NetCDFReader.loadNetCDFNDVar(uri, "lat")
```
Comment (chrismattmann) [inline src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala]: remove commented code

File: src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    //    val numVars = someVars.size()
+    //    var idx = 0
+    //    while (idx < numVars) {
+    //      val variable = someVars.get(idx)
+    //      println(variable)
+    //      idx += 1
+    //    }
+
+    //    val surfTemp = NetCDFReader.loadNetCDFNDVar(uri, variable)
+    //    println("surfTemp: " + surfTemp._2.deep.mkString(" "))
```
Comment (chrismattmann) [inline src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala]: remove commented code

File: src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    /**
+     *  Each SciTensor of this RDD here is of the form T(time,lat,lon)
+     *  where T = surface temperature.
+     */
+    val rdd = sc.NetcdfDFSFile(netcdfDir, variables, partCount)
+
+    //    /** some plausibility checks */
+    //    val rddCollected = rdd.collect()
+    //    println(rddCollected.length)
+    //    val firstTensorShape = rddCollected(0).variables.get("TSURF").map(_.shape)
```
Comment (chrismattmann) [inline src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala]: remove commented code

File: src/main/scala/org/dia/core/SciSparkContext.scala [hunk 5]
```diff
@@ -104,7 +105,8 @@ class SciSparkContext(val conf: SparkConf) {
 
       varName.foreach(y => {
         val arrayandShape = loadNetCDFNDVar(dataset, y)
-        val absT = new BreezeTensor(arrayandShape)
+//        val absT = new BreezeTensor(arrayandShape)
```
Comment (chrismattmann) [inline src/main/scala/org/dia/core/SciSparkContext.scala]: remove commented code

File: src/test/scala/org/dia/tensors/perf/MainMemoryNd4JPerformanceTest.scala [hunk 6]
```diff
@@ -17,33 +17,28 @@
...
 import org.nd4j.linalg.factory.Nd4j
 import org.nd4s.Implicits._
-import org.scalatest.{FunSuite, Ignore}
+import org.scalatest.{ FunSuite, Ignore }
+
 /**
- * The Nd4j Performance Tests
- * Created by rahulsp on 7/7/15.
+ * Nd4j Performance Tests
+ * @author Created by rahulsp on 7/7/15.
```
Comment (chrismattmann) [inline src/test/scala/org/dia/tensors/perf/MainMemoryNd4JPerformanceTest.scala]: side note to us - we should probably remove all author tags from the code @legendOfZelda - thoughts? maybe a new PR/issue.

**Comment by chrismattmann** [comment]
please see my comments thanks @legendOfZelda

File: src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala (hunk 1 above)
Comment (ghost) [inline src/main/scala/org/dia/algorithms/pdfclustering/MainCompute.scala:18]: ok, ill make sure every .scala file has one

File: src/test/scala/org/dia/tensors/perf/MainMemoryNd4JPerformanceTest.scala (hunk 6 above)
Comment (ghost) [inline src/test/scala/org/dia/tensors/perf/MainMemoryNd4JPerformanceTest.scala]: yes. let me remove them all, there's only a few anyway

**Comment by ghost** [comment]
in [https://github.com/legendOfZelda/SciSpark/blob/master/src/main/scala/org/dia/core/SciSparkContext.scala] im now using Nd4jTensor instead of BreezeTensor. to be proper i should use the TensorFactory.getTensor. however, then i have an additional string arg that says whether we want to use Breeze/Nd4j and that arg might have to be propagated all the way up, if we want to make it really proper. but in the end we will decide on one of these two libs anyway...

**Comment by chrismattmann** [comment]
agreed @legendOfZelda

**Comment by chrismattmann** [comment]
merged thanks @legendOfZelda !

**Pull request merged**